go/kbnm/hostmanifest/whitelist.go

```diff
@@ -0,0 +1,57 @@
+package hostmanifest
+
+import (
+	"encoding/json"
+	"os"
+	"path/filepath"
+)
+
+// whitelistPath is used for installing the whitelist as a JSON into a given
+// path on disk.
+type whitelistPath struct {
+	// Root is the path of the machine-global NativeMessage whitelists.
+	Root string
+	// Home is the path of the NativeMessage whitelists for regular users.
+	Home string
+}
+
+func (w *whitelistPath) path(u User, appID string) string {
+	if u.IsAdmin() {
+		return filepath.Join(u.PrefixPath(), w.Root, appID+".json")
```

Comment:
K, I'm going to say this change for a future PR since I already have work based on top of this PR.